README.md

```diff
@@ -1,16 +1,165 @@
-# Retrieval Augmented Generation
+# Databricks Generative AI Cookbook
 
-Please visit http://ai-cookbook.io for the accompanying documentation for this repo.
+Please visit [ai-cookbook.io](http://ai-cookbook.io) for the accompanying documentation for this repository.
 
-This repo provides [learning materials](https://ai-cookbook.io/) and [production-ready code](https://github.com/databricks/genai-cookbook/tree/v0.2.0/agent_app_sample_code) to build a **high-quality RAG application** using Databricks. The [Mosaic Generative AI Cookbook](https://ai-cookbook.io/) provides:
-  - A conceptual overview and deep dive into various Generative AI design patterns, such as Prompt Engineering, Agents, RAG, and Fine Tuning
-  - An overview of Evaluation-Driven development
-  - The theory of every parameter/knob that impacts quality
-  - How to root cause quality issues and detemermine which knobs are relevant to experiment with for your use case
-  - Best practices for how to experiment with each knob
+This repository provides [learning materials](https://ai-cookbook.io/) and code examples to build a **high-quality Generative AI application** using Databricks. The Cookbook provides:
 
-The [provided code](https://github.com/databricks/genai-cookbook/tree/v0.2.0/agent_app_sample_code) is intended for use with the Databricks platform.  Specifically:
-- [Mosaic AI Agent Framework](https://docs.databricks.com/en/generative-ai/retrieval-augmented-generation.html) which provides a fast developer workflow with enterprise-ready LLMops & governance
-- [Mosaic AI Agent Evaluation](https://docs.databricks.com/en/generative-ai/agent-evaluation/index.html) which provides reliable, quality measurement using proprietary AI-assisted LLM judges to measure quality metrics that are powered by human feedback collected through an intuitive web-based chat UI
+- A conceptual overview and deep dive into various Generative AI design patterns, such as Prompt Engineering, Agents, RAG, and Fine-Tuning.
+- An overview of Evaluation-Driven Development.
+- The theory of every parameter/knob that impacts quality.
+- How to root cause quality issues and determine which knobs are relevant to experiment with for your use case.
+- Best practices for how to experiment with each knob.
 
-![Alt text](rag_app_sample_code/dbxquality.png)
\ No newline at end of file
+## TL;DR:
+
+Choose the recipe that best matches your needs:
+
+- **For RAG Applications:**
+  - [RAG Getting Started](./rag_app_sample_code/README.md)
+  - Start with `agent_app_sample_code/A_POC_app` to build a proof of concept
+  - Then explore `agent_app_sample_code/B_quality_iteration` to improve quality
+  - Uses Databricks' Mosaic AI Agent Framework for enterprise features
```

Comment:
what does "enterprise features" mean here? can we be more explicit and say something like "agent serving" and "agent eval"